lldb/test/API/lang/swift/deployment_target/Makefile

```diff
@@ -2,10 +2,12 @@ all: a.out dlopen_module
 
 include Makefile.rules
 
+ARCH := x86_64
+
 # This test only works on macOS 10.11+.
 
 a.out: main.swift libNewerTarget.dylib
-	$(SWIFTC) -target x86_64-apple-macosx10.10 -g -Onone $^ -lNewerTarget -L$(shell pwd) -o $@ $(SWIFTFLAGS) -I$(shell pwd) -Xfrontend -disable-target-os-checking
+	$(SWIFTC) -target $(ARCH)-apple-macosx10.10 -g -Onone $^ -lNewerTarget -L$(shell pwd) -o $@ $(SWIFTFLAGS) -I$(shell pwd) -Xfrontend -disable-target-os-checking
```

Comment:
This is good.

I think the Makefile would safer by doing a recursive make invocation that just sets `SWIFTFLAGS_EXTRAS=-Xfrontend -disable-target-os-checking and TRIPLE=$(ARCH)-apple-macosx10.10 LD_EXTRAS=-lNewerTarget`

```

Reply:
Can we do this as a follow up? It feels like a different improvement.

Reply:
Sure thing. I meant this to say: "the change is fine as is, if you want to go the extra mile, we can remove the meat of the build rules altogether by recursively calling make"

Reply:
Thanks.